api/server/server.go

```diff
@@ -109,7 +110,7 @@ type Config struct {
 func NewServer(conf Config) (*Server, error) {
 	var err error
 	var masterAddr address.Address
-	c, cls, err := lotus.New(conf.LotusAddress, conf.LotusAuthToken)
+	c, cls, err := lotus.New(conf.LotusAddress, conf.LotusAuthToken, lotusConnectionRetries)
```

Comment:
Now we retry some times before considering the Lotus node to be down and abort Powergate start.
Lotus is generally slower to start and enable its API endpoint.